internal/controller/controllers/conditions.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2020.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
+	hivev1 "github.com/openshift/hive/apis/hive/v1"
+)
+
+const (
+
+	//Common
+	SyncedOkReason     string = "SyncOK"
+	SyncedOkMsg        string = "The Spec has been successfully applied"
+	BackendErrorReason string = "BackendError"
+	BackendErrorMsg    string = "The Spec could not be synced due to backend error:"
+	InputErrorReason   string = "InputError"
+	InputErrorMsg      string = "The Spec could not be synced due to an input error:"
+
+	NotAvailableReason string = "NotAvailable"
+	NotAvailableMsg    string = "Information not available"
+
+	ValidationsPassingReason string = "ValidationsPassing"
+	ValidationsUnknownReason string = "ValidationsUnknown"
+	ValidationsFailingReason string = "ValidationsFailing"
+
+	InstalledReason              string = "InstallationCompleted"
+	InstalledMsg                 string = "The installation has completed:"
+	InstallationFailedReason     string = "InstallationFailed"
+	InstallationFailedMsg        string = "The installation has failed:"
+	InstallationNotStartedReason string = "InstallationNotStarted"
+	InstallationNotStartedMsg    string = "The installation has not yet started"
+	InstallationInProgressReason string = "InstallationInProgress"
+	InstallationInProgressMsg    string = "The installation is in progress:"
+	UnknownStatusReason          string = "UnknownStatus"
+	UnknownStatusMsg             string = "The installation status is currently not recognized:"
+
+	//ClusterDeployment Conditions
+	ClusterSpecSyncedCondition hivev1.ClusterDeploymentConditionType = "SpecSynced"
+
+	ClusterInstalledCondition hivev1.ClusterDeploymentConditionType = "Installed"
```

Comment:
Per https://github.com/openshift/hive/pull/1356 would you be ok with changing this one to "Completed"? We went through on Hive team and ended up changing Installed to Compelted to avoid a stutter when we transfer back to ClusterDeployment with a prefix becoming ClusterInstallInstalled. (we felt ClusterInstallCompleted was preferable)